StackExchange.Redis.Resilience/StackExchange.Redis.Resilience.csproj

```diff
@@ -22,7 +22,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="StackExchange.Redis" Version="[2.0.495,2.1)" />
+    <PackageReference Include="StackExchange.Redis"  Version="2.2.62" />
```

Comment:
I would change this to `[2.2.3,2.3)` as it seems that `StackExchange.Redis` introduces breaking changes with minor versions.